app/javascript/common/constants.js

```diff
@@ -0,0 +1,20 @@
+export const MAX_LENGTH_NAMES = 24;
+export const MAX_LENGTH_DESCRIPTIONS = 128;
+
+const maxLengthIs = `Maximum length is %s characters.`;
+const requiredMaxLengthIs = `Required. ${maxLengthIs}`;
+const youHaveReachedMax = `You have reached the maximum length of %s characters.`;
+
+export const validation = {
+  name: {
+    help: sprintf(__(maxLengthIs), MAX_LENGTH_NAMES),
```

Comment:
@mturley The i18n here will not work because the `gettext` parser looks for a `__('string')` format during string extraction.

Can you move the `__(` to the const above?

```js
const maxLengthIs = __('Maximum length is %s characters.');
```